packages/@aws-cdk/aws-ec2/lib/vpn.ts

```diff
@@ -75,6 +92,21 @@ export interface VpnConnectionOptions {
   readonly tunnelOptions?: VpnTunnelOption[];
 }
 
+/**
+ * The VpnGateway Properties
+ */
+export interface VpnGatewayProps extends CfnVPNGatewayProps {
```

Comment:
No exposing L1 types at the L2 level. Copy all the fields over by hand.

Reply:
Done